### ISSUE TITLE
Daily builds?

### ISSUE THREAD

**Issue by shawly** [willfarrell/docker-autoheal#96]
My watchtower constantly updates autoheal because of the daily builds, which is kind of bothersome since I always get notifications for that.

Now I saw that the Dockerfile uses `alpine:3.13.5` hardcoded, so I don't really get why it is built daily when it is always the same EOL alpine image.

I think it would make more sense to either use `alpine:latest` and just build if alpine:latest is updated or just disable daily builds as they don't bring any benefits in the current state.

**Comment by shawly** [comment]
The alpine jq package hasn't been updated since January 2020 https://pkgs.alpinelinux.org/packages?name=jq&branch=v3.13&repo=&arch=&maintainer= and jq itself hasn't seen an update since 2018, so I wouldn't see that as a benefit. 
It's rather luck that the daily builds haven't killed compatibility with the autoheal entrypoint scripts because jq isn't updated regularily so the syntax never changes.

**Comment by MyWay** [comment]
I agree. Would it be possible to move current latest build to nightly or edge?
